lib/stripe/terminal/reader.ex

```diff
@@ -0,0 +1,130 @@
+defmodule Stripe.Terminal.Reader do
+  @moduledoc """
+  A Reader represents a physical device for accepting payment details
+
+  You can:
+  - [Create a Reader](https://stripe.com/docs/api/terminal/readers/create)
+  - [Retrieve a Reader](https://stripe.com/docs/api/terminal/readers/retrieve)
+  - [Update a Reader](https://stripe.com/docs/api/terminal/readers/update)
+  - [Delete a Reader](https://stripe.com/docs/api/terminal/readers/delete)
+  - [List all Readers](https://stripe.com/docs/api/terminal/readers/list)
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+  require Stripe.Util
+
+  @type t :: %__MODULE__{
+          id: Stripe.id(),
+          device_type: String.t(),
+          label: String.t(),
+          location: String.t(),
+          metadata: Stripe.Types.metadata(),
+          serial_number: String.t(),
+          status: String.t(),
+          object: String.t(),
+          device_sw_version: String.t(),
+          ip_address: String.t(),
+          livemode: boolean()
+        }
+
+  defstruct [
+    :id,
+    :device_type,
+    :label,
+    :location,
+    :metadata,
+    :serial_number,
+    :status,
+    :object,
+    :device_sw_version,
+    :ip_address,
+    :livemode
+  ]
+
+  @plural_endpoint "terminal/readers"
+
+  @doc """
+  Create a new reader
+  """
+
+  @spec create(params, Stripe.options()) :: {:ok, t} | {:error, Stripe.Error.t()}
+        when params:
+               %{
+                 :registration_code => String.t(),
+                 :label => String.t(),
+                 :location => Stripe.id(),
```

Comment:
Looks like `label` and `location` are optional?

https://stripe.com/docs/api/terminal/readers/create

Reply:
missed that--thanks. Latest push fixes it.